### DISCUSSION TITLE
error in IDEA(1.1.17.1262)

### DISCUSSION THREAD

**Discussion by WhySoHardToGiveName** [community/community#15499] [Copilot Conversations] [answered]
I updated copilot to version 1.1.17.1262 yesterday, and error occurred as following.
升级插件后出现如下错误
![03d78d55fdd549ede35b95e72be13b8](https://user-images.githubusercontent.com/89514061/164983682-91cc216e-f483-4c5a-a2c8-8d1595ac4ac6.png)

**Comment by l280900756** [comment]
解决了吗？

**Reply by WhySoHardToGiveName** [reply]
没有，试过卸载了再下载还是这样

**Comment by inernoro** [comment]
I have the same bug

**Comment by wapffu** [comment]
我也是这样得 要是解决了记得说一下啊

**Reply by inernoro** [reply]
> 我也是这样得 要是解决了记得说一下啊

还原版本就可以了，我用的1.1.16x版本

**Reply by WhySoHardToGiveName** [reply]
怎么还原版本呀

**Comment by iPisces77** [comment]
建议用降一个版本,已经折腾过各种代理 无效

**Reply by WhySoHardToGiveName** [reply]
求问怎么降版本

**Reply by inernoro** [reply]
首先，去idea下载一个插件包，
![image](https://user-images.githubusercontent.com/9151851/165355894-dcdeb69a-51e6-4ee1-877c-ac585873d598.png)
https://plugins.jetbrains.com/plugin/17718-github-copilot/versions
然后在idea里
![image](https://user-images.githubusercontent.com/9151851/165355804-d7b1e65c-d914-4d28-8512-da835ca3da76.png)
重启就可以了

**Reply by WhySoHardToGiveName** [reply]
可以了！感谢！

**Comment by mings-operation** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/91174807/165416282-3f177b5f-9058-4d3f-8ab6-954f984a740e.png)
![image](https://user-images.githubusercontent.com/91174807/165416333-9eb46090-e5f7-479a-b16a-31192ef5e2fc.png)
选择一下下载
![image](https://user-images.githubusercontent.com/91174807/165416391-e365d438-8a6e-4468-8bb3-c8a69f2c26c6.png)
加载安装包

**Reply by WhySoHardToGiveName** [reply]
ok了！感谢

**Reply by Louaq** [reply]
OK，非常感谢，这个问题终于解决了

**Comment by jansorg** [comment]
Thanks for reporting! We'll investigate this and hope to have a fix soon.

**Comment by jansorg** [comment]
The latest release, 1.1.19, has a fix for this problem. Thank you for your patience!

**Reply by Louaq** [reply]
感谢

**Comment by wapffu** [comment]
我之前买了激活软件 现在找不到了&nbsp;&nbsp;


wapffu
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年4月27日(星期三) 凌晨1:14
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [github/feedback] error in IDEA(1.1.17.1262) (Discussion #15499)





 
首先，去idea下载一个插件包，
 
 https://plugins.jetbrains.com/plugin/17718-github-copilot/versions
 然后在idea里
 
 重启就可以了
